sticker/lib/util.py

```diff
@@ -40,6 +125,54 @@ def convert_image(data: bytes) -> (bytes, int, int):
     return new_file.getvalue(), w, h
 
 
+def _convert_sticker(data: bytes) -> (bytes, str, int, int):
+    mimetype = guess_mime(data)
+    if mimetype.startswith("video/"):
```

Comment:
In the `upload_sticker` function in `sticker/pack.py`, an early return is triggered if the mimetype does not start with "image/" - *before*  `_convert_sticker` is ever called. 

Would be nice to amend that check to allow videos, so sticker creation from e.g. webm is possible (which works just finde wihtout that early return)

Reply:
Yeah, so I not use `return` at here

Reply:
I'm not sure I understand. With how it currently is, you can't add stickers from webm files for example, even though your code works without problems on webm. So it would be nice if in the pack.py, video mime types were also allowed

Reply:
Nevermind,  I misunderstood what you said ,😄